### ISSUE TITLE
Add Internal IP 

### ISSUE THREAD

**Issue by andlyk** [dizzythinks/asavpnusers#4]
Not sure if it is on purpose that the script is not displaying the internal IP or if it can easily be done
The internal IP can be located at OID 1.3.6.1.4.1.9.9.392.1.3.21.1.8

I am hoping for something a long the output of this
Username, Internal IP, Public IP

**Comment by andlyk** [comment]
The internal IP can be located at OID 1.3.6.1.4.1.9.9.392.1.3.21.1.8

**Comment by B3DTech** [comment]
Yes - I'd love to have a few options to include in the output.  Internal IP, AnyConnect Client version, host OS version, connected time.  We use the output of your script in json as an input to our Zabbix monitoring system and field support would love all this data.  I can usually follow SNMP scripts pretty well and modify them myself, but for some reason I can't wrap my mind around yours.  Thanks!

**Comment by dizzythinks** [comment]
" for some reason I can't wrap my mind around yours" < you and me both. Let me think about it as I wrote this three years ago and I don't have access to the kit anymore.

**Comment by MaxDiOrio** [comment]
I [forked](https://github.com/ieeeglobalspec/asavpnusers/commit/76a9017d6be917078d233f64c120e382d385111f) yours and updated it to add the internal IP as well.  Change the output format to yaml as it works better for us, since we're using the script to import the data into Zabbix.
